apps/src/templates/levelSummary/SummaryEntryPoint.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import getScriptData from '@cdo/apps/util/getScriptData';
+import Button from '@cdo/apps/templates/Button';
+import i18n from '@cdo/locale';
+import styles from './summary-entry-point.module.scss';
+
+const SUMMARY_PARAM = 'view=summary';
+
+const SummaryEntryPoint = ({students}) => {
+  const summaryUrlJoin = window.location.search === '' ? '?' : '&';
+  const summaryUrl = window.location.href + summaryUrlJoin + SUMMARY_PARAM;
+
+  const data = getScriptData('freeresponse');
+
+  return (
+    <div className={styles.summaryEntryPoint}>
+      <Button
+        color="neutralDark"
+        text="View student responses"
+        icon="arrow-up-right-from-square"
+        href={summaryUrl}
+        __useDeprecatedTag
```

Comment:
i'm using this flag because the `Button` styles push it down in the page for some reason:
![image](https://user-images.githubusercontent.com/1382374/226976931-88bf36f0-ff83-40f3-bbe9-51cfcb64e899.png)
but if it uses an `<a>` it looks normal. I'm not sure why `<a>` is considered a deprecated tag in this component; unlike `<div>`, this is actually an accessible implementation afaik.

Reply:
This is a known issue -- this tag was added to prevent using a `div` as a button but accidentally discouraged using `a` tags too. :( [receipt](https://github.com/code-dot-org/code-dot-org/pull/44815#discussion_r808387879)